docs/conf.py

```diff
@@ -21,7 +21,12 @@
 author = "The JupyterGIS Development Team"
 language = "en"
 
-jupyterlite_contents = ["../examples/*.jGIS", "../examples/*.json"]
+jupyterlite_contents = [
+    "../examples/*.jGIS",
+    "../examples/*.json",
+    "../examples/nyc.zip",
+    "../examples/radar.gif",
```

Comment:
```suggestion
    "../examples/*.zip",
    "../examples/*.gif",
```

Perhaps we could already include all the files, to avoid any problems if the file names change or if we add files in a future PR?

Reply:
Sure, sounds good to me. I'll also include `.geoJSON` in that case. Thanks :)